File: prime.py
"""def checkPrime(a):
    if a>1:
        for j in range(2,int(a/2)):
            if(a%j)==0:
                print(a, "is not a prime number")
                break
            else:
                print(a, "is a prime number")
                break
                
    else:
        print(a, "is not a prime number")

a=int(input("Enter a number "))
checkPrime(a)
"""

lower=int(input("Enter the lower limit: "))
upper=int(input("Enter the upper limit: "))

for n in range(lower,upper+1):
    if n>1:
        for i in range(2,n):
            if(n%i == 0):
                break
        else:
            print(n)

File: prime.c
#include<stdio.h>

int main()
{
	int n,m=0,i,flag=0;
	printf("Enter the number : ");
	scanf("%d",&n);
	if(n==0 || n==1)
	{
		printf("Number is not a prime\n");
		return 0;
	}
	m=n/2;
	for(i=2;i<m;i++)
	{
		if(n%i==0)
		{
		printf("Number is not a prime\n");
		flag=1;
		break;
		}
	}
	if(flag==0)
	{
		printf("Number is a prime\n");
		return 0;


	}
}
